.gitattributes

```diff
@@ -0,0 +1,26 @@
+# Auto detect text files and perform LF normalization
```

Comment:
Thanks @ericbuehl .  That's what I should have done. I have deleted it from the seDataDevices branch according.

(It took me **ages** to discover why I was getting a **really** obscure error message when I copied working code over to my Raspberry Pi.  It turned out to be from the kernel not recognising the file mentioned in the #! line, due to the existence of  an invisible CR at the end.  And then another eon before I discovered that my git setup was "helpfully" changing LF back to CRLF for me automatically, because it knew I was working on a Windows machine.  So when I finally fathomed all that out, I forgot to investigate where the best place to record the appropriate .gitattributes entries should be)